### PR TITLE
Add known issues to changelog

### LINKED ISSUE #633: Beginners guide should specify k8s version
## Is this a BUG REPORT or FEATURE REQUEST?
 
 Choose one: BUG REPORT

**What happened**:

While following the minikube based approach at http://openebs.readthedocs.io/en/latest/install/dev_solutions.html I ran the `kubectl apply -f openebs-operator.yaml` command. I got the following error:

```
error: error validating "openebs-operator.yaml": error validating data: unknown object type schema.GroupVersionKind{Group:"", Version:"v1", Kind:"ServiceAccount"}; if you choose to ignore these errors, turn validation off with --validate=false
```

**What you expected to happen**:

I would expect it to not produce an error message. However... https://github.com/kubernetes/kubernetes/issues/53309 looks like it bit me.

```bash
 kubectl version
Client Version: version.Info{Major:"1", Minor:"8", GitVersion:"v1.8.1", GitCommit:"f38e43b221d08850172a9a4ea785a86a3ffa3b3a", GitTreeState:"clean", BuildDate:"2017-10-12T00:44:36Z", GoVersion:"go1.9.1", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"7", GitVersion:"v1.7.5", GitCommit:"17d7182a7ccbb167074be7a87f0a68bd00d58d97", GitTreeState:"clean", BuildDate:"2017-09-18T20:30:29Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
```

I'm very new to k8s so I think this is probably a bug from using the bleeding edge stuff. Maybe you should mention in your getting started guide if there is a blessed version of k8s, etc. that is recommended.

### PR DESCRIPTION
**Pull request by mahebbar**
**What this PR does / why we need it**: Known issues added for minikube installation

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #633

**Special notes for your reviewer**:

### CODE REVIEW

File: documentation/source/release_notes/releasenotes.rst [hunk 1]
```diff
@@ -34,6 +34,30 @@ Issues Fixed in v0.4.0
...
+**Issue:**
+
+Setting up OpenEBS with Kubernetes using Minikube on the Ubuntu host displayed the following error.
+*error: error validating "openebs-operator.yaml": error validating data: unknown object type schema.GroupVersionKind{Group:"", Version:"v1", Kind:"ServiceAccount"}; if you choose to ignore these errors, turn validation off with --validate=false*
+
+**Resolution**
+
+1. Download a specific/compatible version, by replacing the **$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt))** portion of the command with a specific version.
+
+   For example, to download version v1.8.0 on MacOS, enter the following command.
```
Comment (yudaykiran) [inline documentation/source/release_notes/releasenotes.rst]: MacOS = Linux

File: documentation/source/release_notes/releasenotes.rst [hunk 2]
```diff
@@ -34,6 +34,30 @@ Issues Fixed in v0.4.0
...
+   ::
+      curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.8.0/bin/da
```
Comment (yudaykiran) [inline documentation/source/release_notes/releasenotes.rst]: Incomplete download URL.

File: documentation/source/release_notes/releasenotes.rst (hunk 1 above)
Comment (mahebbar) [reply documentation/source/release_notes/releasenotes.rst]: Done!

File: documentation/source/release_notes/releasenotes.rst (hunk 2 above)
Comment (mahebbar) [reply documentation/source/release_notes/releasenotes.rst]: Done! My bad :)

File: documentation/source/release_notes/releasenotes.rst [hunk 3]
```diff
@@ -47,9 +47,9 @@ Setting up OpenEBS with Kubernetes using Minikube on the Ubuntu host displayed t
 
 1. Download a specific/compatible version, by replacing the **$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt))** portion of the command with a specific version.
 
-   For example, to download version v1.8.0 on MacOS, enter the following command.
+   For example, to download v1.8.0 on Linux, enter the following command.
    ::
-      curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.8.0/bin/da
+      curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.8.0/bin/darwin/amd64/kubectl
```
Comment (yudaykiran) [inline documentation/source/release_notes/releasenotes.rst]: darwin = linux :)

**Pull request merged**